### PR TITLE
gitflow maven plugin update 1.16.0

### PR DESCRIPTION
**Pull request by dk1844**
The build plugin `com.amashchenko.maven.plugin:gitflow-maven-plugin` has been updated to latest version as-of today, i.e `1.16.0`

### CODE REVIEW

**Review by Zejnilovic — DISMISSED** [review]

File: pom.xml [hunk 1]
```diff
@@ -136,6 +136,7 @@
         <maven.sources.version>3.0.1</maven.sources.version>
         <maven.surefire.version>2.18.1</maven.surefire.version>
         <maven.war.version>2.2</maven.war.version>
+        <gitflow.maven.plugin.version>1.16.0</gitflow.maven.plugin.version>
```
Comment (benedeki) [inline pom.xml:139]:
```suggestion
        <maven.gitflow,plugin.version>1.16.0</maven.gitflow.plugin.version>
```
And they are actually alphabetically ordered. 

Bonus point if you change `<dockerfile.maven.version>` too. 👼

File: pom.xml [hunk 2]
```diff
@@ -416,7 +417,7 @@
             <plugin>
                 <groupId>com.amashchenko.maven.plugin</groupId>
                 <artifactId>gitflow-maven-plugin</artifactId>
-                <version>1.12.0</version>
+                <version>${gitflow.maven.plugin.version}</version>
```
Comment (benedeki) [inline pom.xml:420]:
```suggestion
                <version>${maven.gitflow.plugin.version}</version>
```

**Review by benedeki — CHANGES_REQUESTED** [review]

**Comment by benedeki** [comment]
Just a support plugin upgrade, no need to test extra.

File: pom.xml (hunk 1 above)
Comment (benedeki) [reply pom.xml:139]: It's actually correct by the plugin name.

**Review by benedeki — APPROVED** [review]

**Review by AdrianOlosutean — DISMISSED** [review]

File: pom.xml (hunk 1 above)
Comment (dk1844) [reply pom.xml]: If I am offer an alternative, I have committed a change in the following fashion:
The version property names 
 - now follow the logic of `plugin-artifact-name` -> `plugin.artifact.name.version`
 - are now alphabetically ordered
 - and the unused properties are removed (`maven.compiler.source`, `maven.compiler.target`)

How about that, @benedeki?

**Review by benedeki — APPROVED** [review]
👍

**Pull request merged**